lib/helpers.bash

```diff
@@ -149,10 +150,31 @@ _bash-it-plugins ()
     _bash-it-describe "plugins" "a" "plugin" "Plugin"
 }
 
+_bash-it_pull_and_update_inner() {
+  git pull --rebase &> /dev/null
+  if [[ $? -eq 0 ]]; then
+    echo "Bash-it successfully updated."
+    echo ""
+    echo "Migrating your installation to the latest version now..."
+    _bash-it-migrate
+    echo ""
+    echo "All done, enjoy!"
+    bash-it reload
+  else
+    echo "Error updating Bash-it, please, check if your Bash-it installation folder (${BASH_IT}) is clean."
+  fi
+}
+
 _bash-it_update() {
   _about 'updates Bash-it'
   _group 'lib'
 
+  declare silent
+  for word in $@; do
+    if [[ ${word} == "--silent" || ${word} == "-s" ]]; then
```

Comment:
Should we also add the new `--silent` option to the `bash-it.completion.bash` file? The `update` command is not covered at the moment, since it did not have any options so far.

Reply:
done